packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1053,15 +1053,14 @@ new PrismaClient({
         rootField = action
       }
 
-      // TODO: Replace with lookup map for speedup
       let mapping
-      if (model) {
+      if (model !== undefined) {
```

Comment:
Can it be null? 🤔

Reply:
No, model can either be `string | undefined`. All calls are properly typed here at least 😅